paddle/fluid/framework/parallel_executor.cc

```diff
@@ -252,7 +252,21 @@ ir::Graph *ParallelExecutorPrivate::ApplyMemoryOptimizePass(ir::Graph *graph) {
     VLOG(10) << "buffer_shared_inplace_pass Applied";
   }
 
-  if (build_strategy_.memory_optimize_) {
+  /**
+   * NOTE(zjl): If BuildStrategy.memory_optimize = None in Python,
```

Comment:
`NOTE(zjl)`->`TODO(zengjinle)`? It's easier to find who write these with the full name.

Reply:
Done.